index.html

```diff
@@ -1,14 +1,79 @@
+            </svg>
+        </a>
+        <a href="#" class="text-gray-400 hover:text-gray-500" aria-label="visit our facebook">
+          <svg aria-label="logo facebook" width="23" height="22" viewBox="0 0 23 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M20.7807 0.778697L2.09979 0.883855C1.4593 0.88746 0.94335 1.40925 0.946956 2.04974L1.05212 20.7316C1.05572 21.3712 1.57751 21.8871 2.218 21.8835L12.276 21.8269L12.2302 13.6948L9.49323 13.7102L9.47539 10.541L12.2123 10.5256L12.1992 8.1885C12.1839 5.47604 13.832 3.98974 16.2522 3.97611C17.4115 3.96959 18.4077 4.05061 18.6985 4.08747L18.7144 6.92243L17.0362 6.93275C15.7202 6.94016 15.4691 7.5672 15.4743 8.48419L15.4857 10.508L18.6242 10.4904L18.2335 13.6619L15.5035 13.6772L15.5493 21.8085L20.9007 21.7784C21.5394 21.7748 22.0554 21.253 22.0518 20.6125L21.9466 1.93153C21.943 1.29104 21.4212 0.775092 20.7807 0.778697V0.778697Z" fill="#ABBCD5"/>
+            </svg>
+        </a>
+        <a href="#" class="text-gray-400 hover:text-gray-500" aria-label=" visit our Twitter">
+          <svg aria-label="logo Twitter" width="21" height="18" viewBox="0 0 21 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M20.9622 2.56742C20.1915 2.91476 19.3625 3.15043 18.4916 3.25946C19.3784 2.72158 20.057 1.87338 20.3725 0.865334C19.5432 1.36351 18.6229 1.72745 17.6423 1.92634C16.8527 1.0934 15.7289 0.576468 14.4899 0.583442C11.7084 0.5991 9.679 3.20582 10.3224 5.89636C6.74183 5.73714 3.55773 4.04004 1.41771 1.44542C0.299879 3.38811 0.857505 5.91814 2.7827 7.19007C2.07733 7.17129 1.41125 6.98167 0.829317 6.66206C0.793303 8.65817 2.23442 10.5173 4.30872 10.9213C3.70328 11.0892 3.03939 11.1314 2.36317 11.0057C2.92054 12.7141 4.51828 13.9503 6.40494 13.9747C4.60172 15.405 2.32332 16.0522 0.0363422 15.7955C1.94982 17.0072 4.21917 17.7075 6.65163 17.6938C14.6508 17.6488 19.132 10.8676 18.8249 4.80986C19.6632 4.197 20.3896 3.43428 20.9622 2.56742Z" fill="#ABBCD5"/>
+            </svg>
+        </a>
+      </div>
+    </div>
+    <div class="mt-8">
+      <img src="image/works.png" alt="Person looking at wall with photos" width={800} height={400}/>
```

Comment:
Esto es html, aqui no se usan las `{}` para los valores de los atributos
```suggestion
      <img src="image/works.png" alt="Person looking at wall with photos" width="800" height="400">
```